### DISCUSSION TITLE
Configuring Info Text Color Transitions

### DISCUSSION THREAD

**Discussion by Alkibiades410** [Stellarium/stellarium#3402] [Q&A] [answered]
Under ver. 0.18.1, I somehow found (or stumbled across) a way to adjust when a selected object's text switches between its assigned colors for "night" and "day." 

For some reasons I have remained unable to duplicate this change. I have thus been stuck using the obsolete version for my work on the ancient Athenian (Greek) calendars.

Basically, I would like to upgrade to the latest version(s) of Stellarium, but I need the text to flip colors when the Sun reaches -6.00 alt. instead of 0.00 alt, i.e. when it transitions between (apparent) civil and nautical twilight not when the Sun reaches the mathematical horizon.

I won't inflict the reasons why upon y'all, but any help would be DEEPLY appreciated.

THANKS in advance.
csp

**Comment by alex-w** [comment, marked as answer]
The color of text can be defined in the GUI: F2 -> Service

The text is flip color when atmosphere average luminance becoming higher than daylight label threshold (the latest on you can define via _viewing/sky_brightness_label_threshold_ in `config.ini`)

**Reply by Alkibiades410** [reply]
Perfect!

**Comment by Alkibiades410** [comment]
It took a little trial and error, but adjusting the threshold worked like a charm. 

For those of you, who might be studying the western ancient world (Babylon, Egypt, Greece, Roman etc), the transition from Nautical to Civil Twilight / Civil to Nautical Twilight affected many activities (esp. nautical operations [vessels needed to come to shore before they lost sight of the horizon]). 

For my own research, I discovered that spotting the Moon's first visible crescent, which marked the beginning of many ancient Greek lunisolar calendars, proves far more difficult in actuality as opposed to simulation if the Sun remains above -6.00 Alt. as the Moon descends below the visible horizon.

The Sun's alt. also affects gauging Acronycal and Heliacal risings/ settings of major stars like Arcturus 

The default atmospheric conditions under either Preetham or ShowMySky are fine and use zero light pollution. Then, set sky_brightness_label_threshold = 15.05. The text will now flip when the Sun crosses -6.00 Alt.

Thanks again. This removed a major headache.
csp
